### ISSUE TITLE
Add support for ffmpeg 6

### ISSUE THREAD

**Issue by marillat** [audacity/audacity#4381]
### Your idea

Would be nice if you can add support for ffmpeg 6

### Problem to be solved

Missing support for ffmpeg 6

**Comment by Ryan-Miller-Audacity** [comment]
Verified support for FFMPEG 6 on both Windows and MacOS.

**Comment by SakiPapa** [comment]
Audacity for macOS does not recognize ffmpeg 6.0 libraries

I am using Audacity 3.2.5 on macOS. (M1 PowerBook 13 Pro, Monterey 12.6.5, 16 GB memory)

My ffmpeg library (Arm version)  was installed by Homebrew.

Yesterday after I updated ffmpeg to version 5.1.2 version 6.0,  I got a warning that the library could not be found.

I confirmed that the ffmpeg libraries are installed in 

/opt/homebrew/Cellar/ffmpeg/6.0/.

I restored the old ffmpeg (5.1.2_6) from the TimeMachine backup back to 

/opt/homebrew/Cellar/ffmpeg/5.1.2_6 

and Audacity recognized the 5.1.2 library.

I think the macOS version of Audacity 3.2.5 (Apple Silicon) is not recognize ffmpeg 6.0.

The same result was occured with Audacity 3.3.0-beta1.
